packages/lib/services/rest/routes/notes.ts

```diff
@@ -238,13 +243,14 @@ export async function downloadMediaFile(url: string /* , allowFileProtocolImages
 	}
 }
 
-async function downloadMediaFiles(urls: string[] /* , allowFileProtocolImages:boolean */) {
+async function downloadMediaFiles(urls: string[], shouldTimeoutFaster: boolean) {
 	const PromisePool = require('es6-promise-pool');
 
 	const output: any = {};
 
 	const downloadOne = async (url: string) => {
-		const mediaPath = await downloadMediaFile(url); // , allowFileProtocolImages);
+		const fetchOptions = shouldTimeoutFaster ? { timeout: 1000 * 10, maxRedirects: 7 } : {};
```

Comment:
Could we explicitly pass these fetch options around instead of having this shouldTimeoutFaster option?

Reply:
I can do that too, I just thought that it would look a little weird to have this object as an argument in the extractNoteFromHtml, but maybe that is for the best since we avoid using a weirder parameter (shouldTimeoutFaster).
I'm going to change this too.